src/Datadog.Trace.AspNet/Startup.cs

```diff
@@ -1,16 +1,17 @@
-#if !NETSTANDARD2_0
-
 using System.Web;
+using Datadog.Trace.AspNet;
+
+[assembly: PreApplicationStartMethod(typeof(Startup), "Register")]
 
-namespace Datadog.Trace.ClrProfiler.Integrations
+namespace Datadog.Trace.AspNet
 {
     /// <summary>
-    ///     Used as the target of a PreApplicationStartMethodAttribute on the assembly to load the AspNetHttpModule into the pipeline
+    ///     Used as the target of a PreApplicationStartMethodAttribute on the assembly to load the TracingHttpModule into the pipeline
     /// </summary>
-    public static class AspNetStartup
+    public static class Startup
```

Comment:
Our `Samples.WebForms` project crashes on startup in IISExpress because Owin identifies both `Samples.WebForms.Startup` and `Datadog.Trace.AspNet.Startup` classes as candidate classes because the follow the pattern of `<AssemblyName>.Startup` [Owin Startup Class Detection](https://docs.microsoft.com/en-us/aspnet/aspnet/overview/owin-and-katana/owin-startup-class-detection). I locally changed it to `AspNetStartup` and there were no issues. Can you change this to another appropriate name that is not `Startup`?

Reply:
`HttpApplicationStartup`?

Reply:
Fix in df8dd9c6f14f88edc2195c552c49a2b1d82ce890.